rust/libnewsboat/src/utils.rs

```diff
@@ -445,6 +445,24 @@ pub fn make_title(rs_str: String) -> String {
     }
 }
 
+/// Get the current working directory.
+/// Returns an empty string on error
+pub fn getcwd() -> String {
```

Comment:
So should `getcwd` return `String` or better a [`PathBuf`](https://doc.rust-lang.org/1.24.0/std/path/struct.PathBuf.html)? `String` may easier map to C++ `std::string`, but since it is a path, [`PathBuf`] may be more appropriate from a Rust point of view.